src/Microsoft.PowerShell.Commands.Utility/commands/utility/Measure-Object.cs

```diff
@@ -451,6 +471,18 @@ private bool IsMeasuringGeneric
             }
         }
 
+        /// <summary>
+        /// Does the begin part of the cmdlet.
+        /// </summary>
+        protected override void BeginProcessing()
```

Comment:
Actually, you should call the base class implementation, base.BeginProcessing(), here.

Reply:
Done